unison-src/transcripts-using-base/fix2049.md

```diff
@@ -13,8 +13,6 @@ catcher act =
 
 tests _ =
   [ catcher do
-      match None with Some x -> x
```

Comment:
What was the deal here?

Reply:
It no longer typechecks - this was a test for exception catching. I think it's okay to delete.

Reply:
I tried to break these transcript changes into individual commits. It might be helpful to look at them that way and see the commit message. In this change, the commit message says that it removes a test for catching a missing case since this is handled at compile time now.